day7/kaina/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Saturn</title>
+    <link rel="stylesheet" href="index.css" />
+    <script src="https://kit.fontawesome.com/a62b29f649.js" crossorigin="anonymous"></script>
+
+  </head>
+  <body>
+    <div class="mobile-wrapper">
+      <img
+        class="mobile-background-image"
+        src="Group 91 1.png"
+        alt="mobile image"
+      />
+      <div class="mobile-container">
+        <div class="mobile-logo">
+          <img src="Group 427.png" alt="logo border" />
+          <p>Saturn is best experienced on our mobile app</p>
+        </div>
+        <button>Download Saturn</button>
+      </div>
+    </div>
+    <div class="tablet-wrapper">
```

Comment:
this wrapper is for both tablet and laptop, don't name it tablet-something, use a more generic name

Reply:
changed name to deskblet

Reply:
use a more generic name - meaning: do not mention tablet or desktop